cvm/quantization/dataset.py

```diff
@@ -39,9 +41,39 @@ def load_imagenet(batch_size):
         last_batch='keep',
         num_workers=30)
 
-def load_imagenet_rec(batch_size, input_size=224):
-    rec_val = os.path.expanduser("~/.mxnet/datasets/imagenet/rec/val.rec")
-    rec_val_idx = os.path.expanduser("~/.mxnet/datasets/imagenet/rec/val.idx")
+
+def download_file(filename):
+    if os.path.exists(filename):
+        return
+    filedir, _ = os.path.split(filename);
+    if not os.path.exists(filedir):
+        os.makedirs(filedir)
+    if "imagenet" in filename:
```

Comment:
I suggested adding a parameter indicating the dataset category,  instead of judging by keyword in the filename.